lib/ovirt_metrics/configurator.rb

```diff
@@ -0,0 +1,24 @@
+module OvirtMetrics
+  class Configurator
+    def initialize
+      self.suppress_warnings = false
+      self.connection_specification_name = 'ovirt_metrics' if ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
Do you have a link where you found to do this?  Want to use it as justification in ancestry gem. (I assume you do based on your super standard comment)